packages/docusaurus-utils/package.json

```diff
@@ -23,6 +23,8 @@
     "escape-string-regexp": "^4.0.0",
     "fs-extra": "^9.1.0",
     "gray-matter": "^4.0.2",
+    "intl": "^1.2.5",
+    "intl-locales-supported": "^1.8.12",
```

Comment:
so, it seems deprecated but seems safe in our case 🤪 

https://github.com/formatjs/formatjs/issues/1789

Reply:
Hmm, didn't know about it, but this is a temporary solution until we set v14 as the minimum version of Node.